server/models/article.js

```diff
@@ -0,0 +1,53 @@
+import Utility from '../utilities/utility';
```

Comment:
The correct name of the folder should be `helpers` and the files should be `utils`